src/components/AutoGrid/index.tsx

```diff
@@ -1,33 +1,17 @@
-/** @jsx jsx */
-import { jsx } from "@emotion/core";
-import { Theme } from "types/global";
-import { mq } from "lib/utils";
+import * as React from "react";
+import styled from "lib/styled";
+import Grid, { GridProps } from "components/Grid";
 
-type SpaceScale = 1 | 2 | 3 | 4 | 5 | 6;
+const AutoGrid: React.FC<GridProps> = styled(Grid)();
 
-interface Props {
-  columns?: number[];
-  gap?: SpaceScale;
-}
+const autoGridDefaultProps: GridProps = {
```

Comment:
Why not inline this assignment instead of creating an intermediary const?

Reply:
Yeah I will do that instead